cmd/pbstatuser/main.go

```diff
@@ -20,25 +30,162 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+const ChannelBuffer = 32
+
+type SourceInfo struct {
+	Authentication clients.Authentication `json:"-"`
+
+	SourceID string `json:"resource_id"`
+
+	Identity identity.XRHID `json:"-"`
+}
+
+var (
+	chAws        = make(chan SourceInfo, ChannelBuffer)
+	chAzure      = make(chan SourceInfo, ChannelBuffer)
+	chGcp        = make(chan SourceInfo, ChannelBuffer)
+	chSend       = make(chan kafka.SourceResult, ChannelBuffer)
+	receiverWG   = sync.WaitGroup{}
+	processingWG = sync.WaitGroup{}
+	senderWG     = sync.WaitGroup{}
+)
+
 func processMessage(ctx context.Context, message *kafka.GenericMessage) {
 	logger := ctxval.Logger(ctx)
+
 	// This is just a test to see if Kafka is configured properly in ephemeral
 	logger.Warn().Msg("RECEIVED KAFKA MESSAGE")
```

Comment:
Please drop this, also there was some big comment block by me which can go as well. I meant this as a temporary thing.